app/controllers/tweets_controller.rb

```diff
@@ -1,2 +1,68 @@
 class TweetsController < ApplicationController
+  before_action :set_tweet, only: %i[ show edit update destroy ]
+
+  def index
+    @tweets = Tweet.all
+  end
+
+  def new
+    @tweet = Tweet.new
+  end
+
+  def create
+    @tweet = current_user.tweets.build(tweet_params)
+    if @tweet.save
+      redirect_to tweets_path, notice: "新規Tweetが作成されました"
+    else
+      flash.now[:alert] = @tweet.errors.full_messages.to_sentence
+
+      render :new #new.html.erbに飛んでくれる
+    end
+
+  end
+
+
+  def edit
+    user_check
```

Comment:
```suggestion
    current_user.tweets.where(id: params[:id]).first
```
例えばこんな感じでもやりたいことは実現できそうかな